pkg/runtime/runtime_test.go

```diff
@@ -16,3 +16,8 @@ func (f fakeProvider) DestroySandbox(podName string, namespace string, targets [
 	//TODO: Write tests for DestroySandbox using the fake kubernetes client
 	return
 }
+
+func (f fakeProvider) GetRuntime() string {
```

Comment:
I would like to see tests implemented for this change before giving the :+1:

Reply:
deal